src/contentful/types.ts

```diff
@@ -0,0 +1,105 @@
+export type NebenanLocales = 'de-DE' | 'de' | 'en-US' | 'en' | 'fr-FR' | 'fr' | 'es-ES' | 'es' | 'it-IT' | 'it';
+
+export type ContentfulSpaceDefaults = {
+  id: string;
+  token: string;
+  preview_token: string;
+};
+
+export type ContentfulSpace<T extends string | undefined = undefined> = T extends string
+  ? ContentfulSpaceDefaults & Record<`content_type_${T}`, string>
+  : ContentfulSpaceDefaults;
+
+export type ContentfulRequestQuery = {
+  preview?: boolean,
+  query_string: string,
+};
+
+export type GetQueryRequestReturnValue = undefined | {
```

Comment:
this type is actually based on the first `createRequest` parameter

```ts
export type RequestQuery = Parameters<typeof createRequest>[0];
```

not sure if we should make it more fine-grained. don't think it's necessary since we're just piping it into `createRequest`

https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype